include/benchmark/benchmark.h

```diff
@@ -1349,36 +1351,28 @@ class LambdaBenchmark : public Benchmark {
  public:
   void Run(State& st) override { lambda_(st); }
 
- private:
   template <class OLambda>
```

Comment:
Is this needed to make `std::make_unique<>()` work?

Reply:
yes.  there are clever workarounds (https://seanmiddleditch.github.io/enabling-make-unique-with-private-constructors/) but i didn't think it was necessary to go to such lengths.